packaging/microshift/config.yaml

```diff
@@ -1,35 +1,37 @@
+#!! Do Not Edit
+#!! This is a generated file
+apiServer:
+    # Kube apiserver advertise address to work around the certificates issue when requiring external access using the node IP. This will turn into the IP configured in the endpoint slice for kubernetes service. Must be a reachable IP from pods. Defaults to service network CIDR first address.
```

Comment:
We're going to want to review the comments we are exposing. We need a way to have a comment like this, which is developer facing, separate from the help text we give to admins. The "...work around the certificates issue..." part shouldn't be exposed.

Reply:
Agreed